internal/api/v1beta1/errors.go

```diff
@@ -20,5 +20,6 @@ var (
 	grpcPermissionDenied    = status.Error(codes.PermissionDenied, errors.ErrForbidden.Error())
 	grpcUnauthenticated     = status.Error(codes.Unauthenticated, errors.ErrUnauthenticated.Error())
 
-	ErrEmptyEmailID = errors.New("email id is empty")
+	ErrEmptyEmailID    = errors.New("email id is empty")
+	ErrInvalideEmailID = errors.New("user email is invalid")
```

Comment:
```suggestion
	ErrInvalidEmailID = errors.New("user email is invalid")
```

Reply:
Also seems like this is also not being used. Lets check if we need this.

Reply:
Yes, removed `ErrInvalidEmailID`.